packages/airnode-examples/scripts/deploy-airnode.ts

```diff
@@ -14,10 +14,13 @@ const main = async () => {
     `docker run -it --rm`,
     `--env-file ${secretsFilePath}`,
```

Comment:
You don't need to pass AWS-specific env variables for GCP. It doesn't do any harm but it's going to be inconsistent with the docs which may be confusing.